### DISCUSSION TITLE
Wrong formatting in GitHub Help

### DISCUSSION THREAD

**Discussion by Alan-Liang** [community/community#23738] [New to GitHub]
<p>The code block in <a href="https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners" rel="nofollow noopener">https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners</a> has the wrong language (http) for highlighting.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>Thanks for pointing that out! I’ll let our documentation team know about it!</p>
